dask_sphinx_theme/static/css/style.css

```diff
@@ -9,10 +9,12 @@
   --blue: #1f5aff;
   --dark-blue: #183d72;
   --light-grey: #f6f6f6;
+  --darker-light-grey: #ebeaee;
   --grey: #33363d;
   --dark-grey: #262326;
   --black: #080815;
-  --green: #17955c;
```

Comment:
this green was only used for syntax highlighting and not actually referenced here